elements/pl-code/pl-code.py

```diff
@@ -231,6 +232,9 @@ def render(element_html, data):
         "prevent_select": prevent_select,
     }
 
+    if pl.has_attrib(element, "id"):
+        html_params["id"] = pl.get_string_attrib(element, "id")
```

Comment:
Looking at this again, would it be better to just build the copy text button as an option directly into pl-code? That way, we don't need this id passthrough, and it seems like a common thing to do with a code block anyway.